base/_grid.scss

```diff
@@ -36,5 +32,9 @@
     } @else {
       @extend %tablet__flex-basis--100;
     }
+
+    &.go-column--no-grow {
```

Comment:
This would produce a lot less css if we moved it outside the loop and just made it:
```
.go-column--no-grow {

}
```

The cascade should take over.

Reply:
Good call! I moved it out and down. I also pulled two other modifier classes down into a `modifiers` section.